project/Makefile

```diff
@@ -20,3 +20,8 @@ test:
 
 docs:
 	cd {{ docs_path }} && sphinx-build -W -b html -d _build/doctrees . _build/html
+
+assets:
+	pushd {{ assets_path }} && npm install && popd
```

Comment:
IMO we need to add this to this file, but interpret a new config value/variable if this should be added to the used bundle

@Soullivaneuh what do you think and how should we achive this or is it not necessary?